homeassistant/components/todoist/calendar.py

```diff
@@ -70,6 +70,9 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
     {
         vol.Required(CONF_TOKEN): cv.string,
+        vol.Optional(CONF_PROJECT_WHITELIST, default=[]): vol.All(
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler

Reply:
This will probaly take me some time, as I've not worked with it yet. But I'll give it a try.

Changing this PR to draft in the meantime.